Sources/Hummingbird/HTTP/Cookie.swift

```diff
@@ -51,6 +60,133 @@ public struct Cookie: Sendable, CustomStringConvertible {
     public var httpOnly: Bool { self.properties[.httpOnly] != nil }
     /// The SameSite attribute lets servers specify whether/when cookies are sent with cross-origin requests
     public var sameSite: SameSite? { self.properties[.sameSite].map { SameSite(rawValue: $0) } ?? nil }
+    
+    private static func isValidValue(_ value: String) -> Bool {
+        // RFC 6265 Section 4.1.1: cookie-octet set
+        // Allowed: 0x21, 0x23-0x2B, 0x2D-0x3A, 0x3C-0x5B, 0x5D-0x7E
+        return value.utf8.allSatisfy { byte in
+            (byte == 0x21) ||
+            (0x23...0x2B).contains(byte) ||
+            (0x2D...0x3A).contains(byte) ||
+            (0x3C...0x5B).contains(byte) ||
+            (0x5D...0x7E).contains(byte)
+        }
+    }
+    
+    private static func isValidName(_ name: String) -> Bool {
+        // RFC 2616 Section 2.2: token = 1*<any CHAR except CTLs or separators>
+        // CTLs: 0-31, 127
+        // Separators: ()<>@,;:\"/[]?={} \t
+        let separators: Set<UInt8> = [
+            UInt8(ascii: "("),
+            UInt8(ascii: ")"),
+            UInt8(ascii: "<"),
+            UInt8(ascii: ">"),
+            UInt8(ascii: "@"),
+            UInt8(ascii: ","),
+            UInt8(ascii: ";"),
+            UInt8(ascii: ":"),
+            UInt8(ascii: "\\"),
+            UInt8(ascii: "\""),
+            UInt8(ascii: "/"),
+            UInt8(ascii: "["),
+            UInt8(ascii: "]"),
+            UInt8(ascii: "?"),
+            UInt8(ascii: "="),
+            UInt8(ascii: "{"),
+            UInt8(ascii: "}"),
+        ]
+        // Space is no in the separators, but is added to the CTLs because it's right behind the last CTL
+        // `0x1F` is the last CTL, and space is `0x20`
+        return !name.isEmpty && name.utf8.allSatisfy { byte in
+            (byte >= 0x21 && byte != 127 && !separators.contains(byte))
+        }
+    }
+    
+    /// Create `Cookie`
```

Comment:
Can we extend the comment here to indicate the difference between this and the standard `init`. Also can we add a comment to the `init` to point people to the `validated` version if they want validation.